clients/vault/src/oracle/handler.rs

```diff
@@ -67,6 +71,9 @@ impl ScpMessageActor {
 			ActorMessage::GetPendingProofs { sender } => {
 				let _ = sender.send(self.collector.get_pending_proofs());
 			},
+			ActorMessage::GetScpState { missed_slot } => {
+				overlay_conn.send(StellarMessage::GetScpState(missed_slot.try_into().unwrap())).await;
```

Comment:
This works just like that? I'm just asking because one day we talked about that maybe converting the number to xdr might be necessary but even better if this is not the case.

Reply:
Yes. It works 100% like this. xdr is used behind the scene to pass value via network.